python/pyspark/ml/classification.py

```diff
@@ -902,12 +902,12 @@ class DecisionTreeClassifier(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPred
     >>> from pyspark.ml.linalg import Vectors
     >>> from pyspark.ml.feature import StringIndexer
     >>> df = spark.createDataFrame([
-    ...     (1.0, Vectors.dense(1.0)),
-    ...     (0.0, Vectors.sparse(1, [], []))], ["label", "features"])
+    ...     (1.0, 1.0, Vectors.dense(1.0)),
```

Comment:
My only nit here is that now this shows usage with weights, only, but the weight is 1. Maybe have two quick examples, one without specifying weight and one with a weight != 1?